packages/iota/src/models/payloads/ITaggedDataPayload.ts

```diff
@@ -14,7 +15,7 @@ export interface ITaggedDataPayload extends ITypeBase<5> {
     /**
      * The tag to use to categorize the data.
      */
-    tag: string;
+    tag: HexEncodedString;
 
     /**
      * The index data.
```

Comment:
Shouldn't this also be a HexEncodedString?

Reply:
I think i missed it. Please review the PR and let me know if i miss any other places as well. Thanks.

Reply:
Fixed.